setup.rst

```diff
@@ -37,32 +37,36 @@ In other words, your new app is ready!
 Running your Symfony Application
 --------------------------------
 
-On production, you should use a web server like Nginx or Apache
-(see :doc:`configuring a web server to run Symfony </setup/web_server_configuration>`).
-But for development, it's convenient to use the :doc:`Symfony Local Web Server <setup/symfony_server>`.
+On production, you should use a web server like Nginx or Apache (see
+:doc:`configuring a web server to run Symfony </setup/web_server_configuration>`).
+But for development, it's better to use the :doc:`Symfony Local Web Server <setup/symfony_server>`.
```

Comment:
>  But for development, it's better to use the Symfony Local Web Server

While I can certainly understand the recommendation, it's always better to use a local setup that mimics your production setup, so I disagree on this change. My suggestion would be to leave "convenient" here as it's a nice tool to easily get started

Would it be an idea to guide developers to the next part if they don't want to use the local web server?

Reply:
well, configuring a local nginx/apache with TLS support is a huge pain. By using the Symfony local webserver, you get it for free, which might actually make you closer to prod.

And compared to `bin/console server:run`, the local web server still relies on PHP-FPM so gets closer to prod.

Reply:
That's fair, but it being easier doesn't make it "better" than using the same server you're using on production.

Reply:
it is better than using nginx/apache without HTTPS, I would say.

Reply:
Again, that's debatable: I'd say it's much better to have the same exact stack as you expect in production than to optimize for having TLS in dev mode, but I do see your point as being valid too.

Just the fact that we're having this discussion points to the "better" being a questionable choice of words.